### PR TITLE
Fixing typos

### PR DESCRIPTION
**Pull request by bsipocz**
Some extreme nitpicking of typo fixes and removal of unused imports.

### CODE REVIEW

**Comment by mdboom** [comment]
:+1:

File: astropy/_erfa/core.pyx.templ [hunk 1]
```diff
@@ -6,7 +6,7 @@
 # update it.
 
 """
-This module contains the Cython parts of the ERFA python warppers.  
+This module contains the Cython parts of the ERFA python wrappers.
```
Comment (embray) [inline astropy/_erfa/core.pyx.templ]: Aww, but I liked "warppers" ;)

**Comment by embray** [comment]
Restarted the one Travis build that timed out.

**Comment by astrofrog** [comment]
Wow, how do all these typos get through? Thanks @bsipocz!

File: astropy/_erfa/core.pyx.templ (hunk 1 above)
Comment (bsipocz) [inline astropy/_erfa/core.pyx.templ]: Well, it made some sense, but shouldn't it be spellt with one p only? ;)

**Pull request merged**